Updates for two mc-releases (#1162)
minio/docs: source/reference/minio-mc/mc-replicate-add.rst

      The replication rule synchronizes versioned delete operations, delete markers, and existing objects to the remote MinIO deployment.

      .. versionchanged:: mc RELEASE.2024-03-03T00-13-08Z

>          You can use a configured ALIAS for the ``--remote-bucket`` flag instaed of URL or IP.

Reviewer: ```suggestion
         You can use a configured ALIAS to the ``--remote-bucket`` flag.
```

Reviewer: If you keep what is, `instaed` :)

   :required:

   Specify the credentials, destination deployment, and bucket of the remote location. 
   Value may be an alias/bucket, location based (IP or URL), or path based.

   For example, a URL based target might look like the following:

   .. code-block::

      https://user:secret@myminio.cloudprovider.tld:9001/bucket

   .. versionchanged:: mc RELEASE.2024-03-03T00-13-08Z

>       The ``--remote-bucket`` value may be an :ref:`alias`.

Reviewer: ```suggestion
      The ``--remote-bucket`` supports specifying an existing :ref:`alias <alias>`
```

Not sure if that's the right tag, i think we created a ref somewhere.

Author: It's correct. fixed

   An alias based target might look like the following:

   .. code-block::

>       minio-target/my-bucket

Reviewer: ```suggestion
      --remote-bucket minio-target/my-bucket
```
